src/main/histFileMod.F90

```diff
@@ -318,6 +330,78 @@ subroutine hist_printflds()
        call shr_sys_flush(iulog)
     end if
 
+    ! Print master field list in separate file if namelist variable
+    ! requests it
+    if (masterproc .and. hist_master_list_file) then
+       ! hardwired table column widths
+       width_col_1 = 4  ! column that shows the variable number, nf
+       width_col_2 = 39  ! variable name column
+       width_col_3 = 94  ! long description column
+       width_col_4 = 65  ! units column
+       width_col_sum = width_col_1 + width_col_2 +  &  ! sum of widths plus
+                       width_col_3 + width_col_4 + 3  ! ...3 blank spaces
+
+       ! Convert integer widths to strings for use in format statements
+       ! These write statements are not outputting to files
+       write(str_w_col_1,'(i1)') width_col_1
+       write(str_w_col_2,'(i2)') width_col_2
+       write(str_w_col_3,'(i2)') width_col_3
+       write(str_w_col_4,'(i2)') width_col_4
+       write(str_w_col_sum,'(i3)') width_col_sum
```

Comment:
- [ ] You can avoid duplicating assumptions about the number of digits by using '(i0)' formatting here